composer.json

```diff
@@ -12,7 +12,8 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "~1.0@dev",
+        "php-http/httplug": "dev-master",
```

Comment:
It cause problem for the coverage build, because it use the lowest stable where the TransferException is abstract, but yeah i can maybe avoid creating TransferException :)